src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -62,27 +64,30 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
         </Serif>
       </>
     )
-
-    return (
-      <Row>
-        <Radio
-          mr="1"
-          onSelect={e => {
-            this.setState({ selectedEditionId: edition.__id } as any)
-          }}
-          selected={this.state.selectedEditionId === edition.__id}
-          disabled={!isEcommerceEnrolled}
-        />
-        {editionFragment}
-      </Row>
-    )
+    if (isEcommerceEnrolled) {
+      return (
+        <Row>
+          <Radio
+            mr="1"
+            onSelect={e => {
+              this.setState({ selectedEditionId: edition.__id } as any)
+            }}
+            selected={this.state.selectedEditionId === edition.__id}
+            disabled={!isEcommerceEnrolled}
+          />
+          {editionFragment}
+        </Row>
+      )
+    } else {
+      return <Row>{editionFragment}</Row>
+    }
   }
   renderEditions() {
     const editions = this.props.artwork.edition_sets
     const editionsFragment = editions.map((edition, index) => {
       return (
         <React.Fragment key={edition.__id}>
-          <Box p={2}>{this.renderEdition(edition)}</Box>
+          <Box p={2}>{this.renderEdition(this.props.artwork, edition)}</Box>
```

Comment:
#minor Since this is a `class` maybe pull `artwork` and `addition` from `this.props` from within `renderEdition`?

Reply:
I think I can do that for `artwork` but I think the edition might need to come through the map iterator in `renderEditions`.